app/react/src/server/cra-config.js

```diff
@@ -83,8 +83,13 @@ export function getCraWebpackConfig(mode) {
 
   const craWebpackConfig =
     mode === 'production' ? 'config/webpack.config.prod' : 'config/webpack.config.dev';
+  
+  const craWebpackConfigFile = path.join(pathToReactScripts, craWebpackConfig);
 
-  let pathToWebpackConfig = require.resolve(path.join(pathToReactScripts, craWebpackConfig));
+  let pathToWebpackConfig;
+  if (fs.existsSync(craWebpackConfigFile)) {
```

Comment:
at this point `craWebpackConfigFile` is not a real path and fs can't really find it. I think that's why all the tests are failing.